lib/components/proposal/ProposalDetails.tsx

```
@@ -247,11 +257,49 @@ function Comments({ proposalIndex, proposalType }: { proposalIndex: string; prop
+    const total = allComments.length;
+
+    const count: Record<ECommentSentiment, number> = {
+      [ECommentSentiment.AGAINST]: 0,
+      [ECommentSentiment.SLIGHTLY_AGAINST]: 0,
+      [ECommentSentiment.NEUTRAL]: 0,
+      [ECommentSentiment.SLIGHTLY_FOR]: 0,
+      [ECommentSentiment.FOR]: 0,
+    };
+
+    for (const comment of allComments) {
+      if (comment.aiSentiment && count.hasOwnProperty(comment.aiSentiment)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)